vaadin-board-flow/src/main/java/com/vaadin/flow/component/board/internal/FunctionCaller.java

```diff
@@ -40,7 +40,7 @@ public static void callOnceOnClientReponse(Component component,
         }
         element.setProperty(trackingProperty, true);
 
-        element.callFunction(function);
```

Comment:
Let's split this PR to only include this line so that we could release a patch version.

IMO it would make sense to remove `HtmlImport` usage in a separate PR after that.

Reply:
Just include the api change doesn't fix the problem, as @HtmlImport has been removed from flow 3.0. 

the current validation build failed, because it is running against flow 2.X-SNAPSHOT, but not 3.0-snapshot. 
we need some more work to make a proper fix

Reply:
Removing the @HtmlImport makes this a breaking change, so you'll also need to include some version bumps to pom.xml as well.

Reply:
for replacing the deprecated API, https://github.com/vaadin/vaadin-board-flow/pull/111

Reply:
> Removing the @HtmlImport makes this a breaking change, so you'll also need to include some version bumps to pom.xml as well.

this is the goal at the end.. I think for V15, we will need all the new major versions of components right?

Reply:
Yes